src/components/molecules/definition/definition.jsx

```diff
@@ -37,6 +39,19 @@ const Definition = ({ results }) => {
 		setDefToggleClassName('definition__show-full');
 	};
 
+	const handleMoreInfoClick = (payload) => {
```

Comment:
Do we need to do an e.preventDefault here and return true at the end to make sure that this executes before it goes to the page?  also, I don't think you need to pass payload